config.json

```diff
@@ -653,8 +653,8 @@
         "slug": "leap",
         "name": "Leap",
         "uuid": "8ba15933-29a2-49b1-a9ce-70474bad3007",
-        "practices": [],
-        "prerequisites": [],
+        "practices": ["math-operators", "if-statements"],
+        "prerequisites": ["math-operators", "if-statements"],
```

Comment:
We don't yet have a Concept Exercise that unlocks `math-operators` . This means that for the website, this prerequisite will be completely ignored. I was wondering if it would make sense to guard against these types of situations by being explicit and also adding `numbers` as the prerequisite. What do you think?

Reply:
I thought the mandate was to create concepts for practice exercises even where there was no concept exercise illustrating the concept.  Did I misunderstand?

Issue #1499 addresses the question of concepts without illustrative exercises.  So far we need concepts for `lambdas`, `enumerables`, `math-operators` and `yield` - after looking at 3 exercises!

Issue #1459 (mismatch between _exercises.json_ and _config.json_) also relates to this.  I think it will indeed cause `math-operators` to be folded into `numbers`.  I hesitated to do it in this PR as the links in the `numbers`'s _about.md_ do not include `%` so text will need to be altered as part of issue #1459.

This is what I mean about keeping things straight.

Reply:
> I thought the mandate was to create concepts for practice exercises even where there was no concept exercise illustrating the concept. Did I misunderstand?

No you understood correctly. This is all just trial and error. I'm just thinking of the effect it would have one students for the time being while we don't have a `math-operators` exercise. That is why I suggested adding the `numbers` prerequisite, althought that would be a slightly duplicate prerequisite once we have `math-operators`.

Reply:
Fair enuf.  `numbers` in addition.  n.p.